libbeat/keystore/keystore.go

```diff
@@ -124,3 +95,23 @@ func ResolverWrap(keystore Keystore) func(string) (string, parse.Config, error)
 		return string(v), parse.DefaultConfig, nil
 	}
 }
+
+// AsWritableKeystore casts a keystore to WritableKeystore, returning an error if the given keystore does not implement
+// WritableKeystore interface
```

Comment:
nit; let's mention that the `ErrNotWritable` error is returned. So devs now what to expect in case they want to check the error value.